Sources/Fullscreen/Consent/Settings/Demo/ConsentViewDemoView.swift

```diff
@@ -0,0 +1,75 @@
+
+//
+//  Copyright © 2018 FINN AS. All rights reserved.
+//
+import FinniversKit
+
+struct Section {
+    var title: String
+    var items: [ConsentViewCellModel]
+}
+
+class ConsentViewDemoView: UIView {
+    
+    private let sections = [Section(title: "Varslinger", items: [ConsentViewCellModel(title: "Prisnedgang på torget", hairLine: false)]),
+
+                            Section(title: "Personvern", items: [ConsentViewCellModel(title: "Få nyhetsbrev fra FINN", stateText: "Av"),
+                                                                 ConsentViewCellModel(title: "Personlin tilpasset FINN", stateText: "På"),
+                                                                 ConsentViewCellModel(title: "Motta viktig informasjon fra FINN", stateText: "På"),
+                                                                 ConsentViewCellModel(title: "Smart reklame"),
+                                                                 ConsentViewCellModel(title: "Last ned dine data"),
+                                                                 ConsentViewCellModel(title: "Slett meg som bruker", hairLine: false)])]
+
+    private lazy var consentView: ConsentView = {
+        let view = ConsentView(frame: .zero, style: .grouped)
```

Comment:
Doesn't this need the `translateAutoresizingEFJIEWFJIOEFJOIE = false`, thingie?

Reply:
The fillInSuperview method does set it to false

Reply:
ahh, that explains everything